<!DOCTYPE html>
<html lang="en">

<!-- Head tag -->
<head>

    <meta charset="utf-8"/>
    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
    <meta name="viewport" content="width=device-width, initial-scale=1"/>

    <!--Description-->

    

    
        <meta name="description" content="Biological PartDNA PCRAs the starting step, human proinsulin coding gene (HPI) was synthesized. Specific primers were synthesized to ligate HPI gene i"/>
    

    <!--Author-->
    
        <meta name="author" content="Will Wu"/>
    

    <!--Open Graph Title-->
    
        <meta property="og:title" content="RESULTS"/>
    

    <!--Open Graph Description-->
    
        <meta property="og:description" content="Biological PartDNA PCRAs the starting step, human proinsulin coding gene (HPI) was synthesized. Specific primers were synthesized to ligate HPI gene i"/>
    

    <!--Open Graph Site Name-->
        <meta property="og:site_name" content="Will Wu&#39;s blog"/>

    <!--Type page-->
    
        <meta property="og:type" content="article"/>
    

    <!--Page Cover-->
    
    
        <meta property="og:image" content="http://yoursite.comhttp://www.codeblocq.com/assets/projects/hexo-theme-clean-blog/img/home-bg.jpg"/>
    

        <meta name="twitter:card" content="summary_large_image"/>

    

    
        <meta name="twitter:image" content="http://yoursite.comhttp://www.codeblocq.com/assets/projects/hexo-theme-clean-blog/img/home-bg.jpg"/>
    

    <!-- Title -->
    
    <title>RESULTS - Will Wu&#39;s blog</title>

    <!-- Bootstrap Core CSS -->
    <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet"/>

    <!-- Custom CSS -->
    
<link rel="stylesheet" href="/css/style.css">


    <!-- Custom Fonts -->
    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" type="text/css"/>
    <link href="//fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic" rel="stylesheet" type="text/css"/>
    <link href="//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800" rel="stylesheet" type="text/css"/>

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
    <script src="//oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
    <script src="//oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

    <!-- Gallery -->
    <link href="//cdnjs.cloudflare.com/ajax/libs/featherlight/1.3.5/featherlight.min.css" type="text/css" rel="stylesheet"/>

    <!-- Google Analytics -->
    


    <!-- favicon -->
    

<meta name="generator" content="Hexo 4.2.0"></head>


<body>

    <!-- Menu -->
    <!-- Navigation -->
<nav class="navbar navbar-default navbar-custom navbar-fixed-top">
    <div class="container-fluid">
        <!-- Brand and toggle get grouped for better mobile display -->
        <div class="navbar-header page-scroll">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/">Configurable Title</a>
        </div>

        <!-- Collect the nav links, forms, and other content for toggling -->
        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
            <ul class="nav navbar-nav navbar-right">
                
                    <li>
                        <a href="/">
                            
                                Home
                            
                        </a>
                    </li>
                
                    <li>
                        <a href="/archives">
                            
                                Archives
                            
                        </a>
                    </li>
                
                    <li>
                        <a href="/tags">
                            
                                Tags
                            
                        </a>
                    </li>
                
                    <li>
                        <a href="https://github.com/WillWu12138/WillWu12138.github.io" target="_blank" rel="noopener">
                            
                                <i class="fa fa-github fa-stack-2x"></i>
                            
                        </a>
                    </li>
                
            </ul>
        </div>
        <!-- /.navbar-collapse -->
    </div>
    <!-- /.container -->
</nav>

    <!-- Main Content -->
    <!-- Page Header -->
<!-- Set your background image for this header in your post front-matter: cover -->

<header class="intro-header" style="background-image: url('http://www.codeblocq.com/assets/projects/hexo-theme-clean-blog/img/home-bg.jpg')">
    <div class="container">
        <div class="row">
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                <div class="post-heading">
                    <h1>RESULTS</h1>
                    
                    <span class="meta">
                        <!-- Date and Author -->
                        
                        
                            2019-12-31
                        
                    </span>
                </div>
            </div>
        </div>
    </div>
</header>

<!-- Post Content -->
<article>
    <div class="container">
        <div class="row">

            <!-- Tags and categories -->
           
                <div class="col-lg-4 col-lg-offset-2 col-md-5 col-md-offset-1 post-tags">
                    
                        


<a href="/tags/Project/">#Project</a>


                    
                </div>
                <div class="col-lg-4 col-md-5 post-categories">
                    
                </div>
            

            <!-- Gallery -->
            

            <!-- Post Main Content -->
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                <h3 id="Biological-Part"><a href="#Biological-Part" class="headerlink" title="Biological Part"></a>Biological Part</h3><p>DNA PCR<br>As the starting step, human proinsulin coding gene (HPI) was synthesized. Specific primers were synthesized to ligate HPI gene into the plasmid pHY-P43 with the two restriction sites using BamHI and EcoRI . The gene in this plasmid is of the length of 345 bp after successful ligation, as shown in the gel electrophoresis image (Fig. 1).</p>
<p><img src="/2019/12/31/RESULTS/1.png" alt="1"></p>
<p>Fig.1 Gel electrophoresis image of the bands for HPI genes</p>
<p>Plasmid Construction<br>After the HPI gene was inserted into pHY-P43 plasmid, The length of the plasmid with HPI gene is 5.5 K bp. The electrophoresis image below (Fig. 2) confirmed that the recombinant plasmid was constructed successfully.</p>
<p><img src="/2019/12/31/RESULTS/2.png" alt="2"></p>
<p>Fig.2 Gel electrophoresis image of the bands for pHY-P43 after double digestion. 1, 3, 4, and 5 are positive results with bands in the region of 500 bp ~ 200 bp which is the length of HPI gene (345 bp) and bands in the region of 5000 bp which is the length of the plasmid. 2 is a negative result. There’s no band in the region of 500 bp ~ 200 bp.</p>
<p>Gene Transformation<br>The plasmid was inserted into E. coli for replication and storage. Colony PCR was implemented to confirm successful transformation of HPI gene into the host E. coli cells. Electrophoresis image (Fig. 3) demonstrates the presence of a gene of length 250~500 bp, which is the proinsulin gene, in the E. coli constructs.</p>
<p><img src="/2019/12/31/RESULTS/3.png" alt="3"></p>
<p>Fig.3 Gel electrophoresis image of the bands for E.coli’s. 1, 2, 3, 4, and 5 are all positive results with bands in the region of 500 bp ~ 200 bp which is the length of HPI gene (345 bp). Thus, they all contained the HPI gene.</p>
<p>To ensure that the gene between length 250~500 bp was HPI instead of some other gene, the band was extracted from the gel and sequenced. The results show that it matches the sequence of the HPI genes we originally used for synthesis. </p>
<p>Genetically engineered E. coli strains bearing HPI gene were cultured overnight. The recombinant plasmid was extracted from E. coli and transformed into B. subtilis WB800N. We presumed that the modified B. subtilis is able to produce proinsulin. Three different validations were performed.</p>
<ol>
<li>Colony PCR<br>Colony PCR for B. subtilis was carried out to ensure that it contains HPI gene from E. coli. The gel electrophoresis image (Fig. 4) confirms that B. subtilis did have the target gene.</li>
</ol>
<p><img src="/2019/12/31/RESULTS/4.png" alt="4"></p>
<p>Fig.4 Gel image of bands for HPI gene within B. subtilis</p>
<ol start="2">
<li>Protein Electrophoresis<br>B. subtilis was cultured for two days. Then all the proteins from the bacteria were extracted to test for proinsulin by protein electrophoresis. Human proinsulin has a molecular weight of about 12 kDa, so there should be an additional band between 14 and 10.5 kDa on the protein electrophoresis image, comparing to the control group that didn’t contain any HPI genes (1 and 4 on Fig.5). Despite numerous attempts, we couldn’t obtain a clear picture showing that particular band.</li>
</ol>
<p><img src="/2019/12/31/RESULTS/5.png" alt="5"></p>
<p>Fig.5 Results from protein electrophoresis before spacer peptide is added. 1 is the control group where the bacteria doesn’t contain HPI gene. 2 and 3 are the experimental group where the bacteria contain HPI gene. 4, 5, and 6 are 10x dilutions of 1, 2, and 3 respectively. There is no visible difference between the bands of the control group and experimental group, meaning that the engineered bacteria didn’t produce HPI as we have expected.</p>
<p>After rigorously analyzing our previous experiments for any mistakes, we found that the inability to produce the band on the gel image was because low expression of proinsulin. Although a strong promoter was used, the quantity of proinsulin obtained was still low. It is evident that the expression efficiency has to be increased. After a careful consideration, we decided to add a spacer peptide on the upstream region of the HPI gene. The results indicate a visible additional band between 14 kDa and 10.5 kDa, comparing to the control group (1 and 4 on Fig. 6). It is seen that the spacer peptide has actually enhance the gene expression, and the bacteria synthesized our desired protein, proinsulin.</p>
<p><img src="/2019/12/31/RESULTS/6.png" alt="6"></p>
<p>Fig.6 Protein electrophoresis after spacer peptide is added. 1 is the control group where the bacteria doesn’t contain HPI gene. 2 and 3 are the experimental group where the bacteria contain HPI gene. 4, 5, and 6 are 10x dilutions of 1, 2, and 3 respectively. There are bands of 5 and 6 with deep color between 14kDa band (grey) and 10.5 kDa band (dark grey) of marker which 4 lacks. This shows that the bacteria do produce proinsulin.</p>
<ol start="3">
<li>Microfluidic Analysis<br>The last approach is based on our microfluidics model (Fig. 7), composed of an organ-on-a-chip, dual channel electrochemical sensors, home-made data acquisition board, and our biological parts. If B. subtilis produce sufficient proinsulin, the liver cells grown on the organ chip should absorb glucose, and lower down the glucose concentration. As a result, the sensors should detect a decrease in current, which is directly proportional to glucose level. More details can be seen in the following organ-on-a-chip section.</li>
</ol>
<p><img src="/2019/12/31/RESULTS/7.png" alt="7"></p>
<p>Fig.7 Data acquisition board with the biochemical sensors inside the organ-on-a-chip</p>
<p>Electrochemical Sensor<br>Dual channel electrochemical sensors were made to monitor blood glucose level. (Fig.8)</p>
<p><img src="/2019/12/31/RESULTS/8.png" alt="8"></p>
<p>Fig.8 Our finished sensors connected to wires</p>
<p>To test the performance of the sensors, electric potential was altered between the range -0.6V to 0.6V on the working electrode of the sensors. A current was produced against the reference electrode which was 0V all the time. The current was measured continuously, and the electric potential against current was plotted below in Fig. 9. The change of current, 0.8 mA to -1.8 mA, on the graph with respect to electric potential shows that the electrodes on the sensors were functional, and the medium on the sensors were conductive.</p>
<p><img src="/2019/12/31/RESULTS/9.png" alt="9"></p>
<p>Fig.9 Cyclic voltammetry graph of the electrodes on the sensor. Current changes with respect to electric potential</p>
<p>After function of the electrodes was confirmed, testing on the sensor was done to demonstrate its overall operationality. The sensor was inserted into a glucose solution of concentration 0.1 mM/L. After an interval of 50s, 5 µL of glucose is added into the solution. The sensor measured the current output. Fig. 10 showed a clear trend between current and time. As time increases, more glucose was added into the solution, so the current increased, from 0 µA to 195 µA. Thus, the sensors have the capability to detect glucose concentration in the whole range since there is clearly a relationship between the glucose concentration and the current measured by the sensor.</p>
<p><img src="/2019/12/31/RESULTS/10.png" alt="10"></p>
<p>Fig.10 Graph showing the overall rise of current measured by the sensor after a series of nine addition of glucose</p>
<p>Correlation between glucose concentration and current was then made. 12 readings were taken for 12 different standard solutions. The glucose concentration of each of the solutions is shown below on the graph (Fig. 11). We applied linear regression to the data and made the line of best-fit which is I=75.43c+1.31. The correlation coefficient is 0.976. This means that the equation constructed should be very close to the actual value according to mathematical calculations. Also, it indicted that the sensors were precise.</p>
<p><img src="/2019/12/31/RESULTS/11.png" alt="11"></p>
<p>Fig.11 Relationship between current and glucose concentration</p>
<p>To confirm that it’s accurate to deduce the unknow glucose concentration just from the current data by using the equation, c = ( I - 1.31 ) / 75.43, four test of known concentration of glucose solution (0.2 mmol/L, 0.4 mmol/L, 0.8 mmol/L, 1.6 mmol/L) were implemented. The graph below was obtained (Fig. 12).</p>
<p><img src="/2019/12/31/RESULTS/12.png" alt="12"></p>
<p>Fig.12 A validation of our equation. The sensor is dipped into known concentration of glucose. The current is plotted out.</p>
<p>In order to make sure that two glucose concentration sensors can work side by side, two sensors were dipped into two solutions (a 2mM glucose solution and a 1x PBS solution) (Fig. 13). The current outputs (Fig. 14) from the two sensors were different. The reading for the sensor to the glucose solution was in the range of 3838<del>3508 mV, while the other one was in the range of 2732</del>2683 mV. The difference indicates that our sensors are capable of working together at the same time and still obtaining accurate values.</p>
<p><img src="/2019/12/31/RESULTS/13.png" alt="13"></p>
<p>Fig.13 A demonstration of the test carried out. One sensor is dipped into 2mM glucose solution whereas the other is inserted into 1x PBS solution.</p>
<p><img src="/2019/12/31/RESULTS/14.png" alt="14"></p>
<p>Fig.14 The current data from the two sensors of the above test. It’s easy to see that they are different.</p>
<h3 id="Organ-on-a-Chip"><a href="#Organ-on-a-Chip" class="headerlink" title="Organ-on-a-Chip"></a>Organ-on-a-Chip</h3><p>The final part of our experiment was performed. The organ chip was fabricated (Fig. 15) and incorporated with the functional biochemical sensors (Fig. 16). Several tests on the chip were carried out before the bacteria was injected.</p>
<p><img src="/2019/12/31/RESULTS/15.png" alt="15"></p>
<p>Fig.15 Our home-made organ-on-a-chip</p>
<p><img src="/2019/12/31/RESULTS/16.png" alt="16"></p>
<p>Fig.16 Integration of organ-on-a-chip with sensors</p>
<h3 id="Test-of-the-blank-chambers"><a href="#Test-of-the-blank-chambers" class="headerlink" title="Test of the blank chambers"></a>Test of the blank chambers</h3><p>First, diffusion rate of glucose inside the organ on chip was tested. The two chambers were filled with PBS. Glucose was injected into the upper chamber (simulating the blood stream). After the current readings of the top and bottom chamber stabilized, another batch of glucose was added. The two sensors on the upper and lower chamber continuously recorded current. From the graph (Fig. 17), it is seen that after 40-80 seconds, the glucose concentration became relatively equal in the two chambers. It can be inferred that little time was taken for glucose to achieve equilibrium between the top and bottom chamber, and glucose can diffuse very easily in between.</p>
<p><img src="/2019/12/31/RESULTS/17.png" alt="17"></p>
<p>Fig.17 The current of the top and bottom chamber before and after glucose addition into the upper chamber</p>
<p>In order to mimic the liver, cancerous liver cell lines, hepatocytes, were transplanted onto the lower chamber of our organ chip (Fig. 18). Two fluorescent dyes, DAPI and CDFDA, were used to assess the liver cell growth. After 48 hours, several images (Fig. 19) were taken using fluorescence microscope. The abundance of blue-colored nucleus, stained by DAPI, in these pictures confirm that there were plenty of viable liver cells present in the chip. Here, the red color in the surroundings signifies MRP2 activity for the hepatocyte efflux transporter. This means that liver cells are functioning normally.</p>
<p><img src="/2019/12/31/RESULTS/18.png" alt="18"></p>
<p>Fig.18 Liver cells injected into the organ chip</p>
<p><img src="/2019/12/31/RESULTS/19.png" alt="19"></p>
<p>Fig.19 Fluorescent image of the liver cells in our organ-on-a-chip. The blue part represents cell nucleus while the red color indicates MRP2 activity.</p>
<p>With the assurance of liver cells growing finely, the same test of glucose diffusion was carried out again. It took much longer time for the stabilization of glucose concentration in the lower chamber, on average 80 seconds (Fig. 20). We concluded that the liver cell impeded the diffusion of glucose, thus the time for equilibrium increased.</p>
<p><img src="/2019/12/31/RESULTS/20.png" alt="20"></p>
<p>Fig.20 The current of the top and bottom chamber containing liver cells before and after glucose addition</p>
<p>In our last experiment, proinsulin-bearing B. subtilis cells were injected into the organ chip. After 48 hours, lysozyme was added to release the proinsulin. The recorded time for glucose equilibrium was about 90 seconds (Fig. 21), longer than previous experiment without proinsulin-producing B. subtilis. Also, the glucose concentration (current) for the bottom chamber was significantly less than that of the top chamber. The above results provide convincing evidence that the release of proinsulin triggered the liver cells to take in glucose. The glucose level was lowered down within our in vitro model by the proinsulin from engineered B. subtilis, The objective of our project to simulate bacterial supply of insulin in vitro for the treatment of diabetes is achieved.</p>


                
            </div>

            <!-- Comments -->
            
                <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                    


                </div>
            
        </div>
    </div>
</article>

    <!-- Footer -->
    <hr />

<!-- Footer -->
<footer>
    <div class="container">
        <div class="row">
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                <ul class="list-inline text-center">
                    

                    

                    
                        <li>
                            <a href="https://github.com/WillWu12138/WillWu12138.github.io" target="_blank">
                                <span class="fa-stack fa-lg">
                                    <i class="fa fa-circle fa-stack-2x"></i>
                                    <i class="fa fa-github fa-stack-1x fa-inverse"></i>
                                </span>
                            </a>
                        </li>
                    

                    

                    

                    
                </ul>
                <p class="copyright text-muted">&copy; 2020 Will Wu<br></p>
                <p class="copyright text-muted">Original Theme <a target="_blank" href="http://startbootstrap.com/template-overviews/clean-blog/">Clean Blog</a> from <a href="http://startbootstrap.com/" target="_blank">Start Bootstrap</a></p>
                <p class="copyright text-muted">Adapted for <a target="_blank" href="https://hexo.io/">Hexo</a> by <a href="http://www.codeblocq.com/" target="_blank">Jonathan Klughertz</a></p>
            </div>
        </div>
    </div>
</footer>


    <!-- After footer scripts -->
    
<!-- jQuery -->
<script src="//code.jquery.com/jquery-2.1.4.min.js"></script>

<!-- Bootstrap -->
<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>

<!-- Gallery -->
<script src="//cdnjs.cloudflare.com/ajax/libs/featherlight/1.3.5/featherlight.min.js" type="text/javascript" charset="utf-8"></script>

<!-- Disqus Comments -->



</body>

</html>