<!DOCTYPE html>
<html lang="en">

<!-- Head tag -->
<head>

    <meta charset="utf-8"/>
    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
    <meta name="viewport" content="width=device-width, initial-scale=1"/>

    <!--Description-->

    

    
        <meta name="description" content="Characterization of Basic PartsSeveral promoters with high expression level that are commonly used in Bacillus subtilis were screened to determine the"/>
    

    <!--Author-->
    
        <meta name="author" content="Will Wu"/>
    

    <!--Open Graph Title-->
    
        <meta property="og:title" content="PARTS"/>
    

    <!--Open Graph Description-->
    
        <meta property="og:description" content="Characterization of Basic PartsSeveral promoters with high expression level that are commonly used in Bacillus subtilis were screened to determine the"/>
    

    <!--Open Graph Site Name-->
        <meta property="og:site_name" content="Will Wu&#39;s blog"/>

    <!--Type page-->
    
        <meta property="og:type" content="article"/>
    

    <!--Page Cover-->
    
    
        <meta property="og:image" content="http://yoursite.comhttp://www.codeblocq.com/assets/projects/hexo-theme-clean-blog/img/home-bg.jpg"/>
    

        <meta name="twitter:card" content="summary_large_image"/>

    

    
        <meta name="twitter:image" content="http://yoursite.comhttp://www.codeblocq.com/assets/projects/hexo-theme-clean-blog/img/home-bg.jpg"/>
    

    <!-- Title -->
    
    <title>PARTS - Will Wu&#39;s blog</title>

    <!-- Bootstrap Core CSS -->
    <link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" rel="stylesheet"/>

    <!-- Custom CSS -->
    
<link rel="stylesheet" href="/css/style.css">


    <!-- Custom Fonts -->
    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" type="text/css"/>
    <link href="//fonts.googleapis.com/css?family=Lora:400,700,400italic,700italic" rel="stylesheet" type="text/css"/>
    <link href="//fonts.googleapis.com/css?family=Open+Sans:300italic,400italic,600italic,700italic,800italic,400,300,600,700,800" rel="stylesheet" type="text/css"/>

    <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
    <!--[if lt IE 9]>
    <script src="//oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
    <script src="//oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
    <![endif]-->

    <!-- Gallery -->
    <link href="//cdnjs.cloudflare.com/ajax/libs/featherlight/1.3.5/featherlight.min.css" type="text/css" rel="stylesheet"/>

    <!-- Google Analytics -->
    


    <!-- favicon -->
    

<meta name="generator" content="Hexo 4.2.0"></head>


<body>

    <!-- Menu -->
    <!-- Navigation -->
<nav class="navbar navbar-default navbar-custom navbar-fixed-top">
    <div class="container-fluid">
        <!-- Brand and toggle get grouped for better mobile display -->
        <div class="navbar-header page-scroll">
            <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                <span class="sr-only">Toggle navigation</span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
                <span class="icon-bar"></span>
            </button>
            <a class="navbar-brand" href="/">Configurable Title</a>
        </div>

        <!-- Collect the nav links, forms, and other content for toggling -->
        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
            <ul class="nav navbar-nav navbar-right">
                
                    <li>
                        <a href="/">
                            
                                Home
                            
                        </a>
                    </li>
                
                    <li>
                        <a href="/archives">
                            
                                Archives
                            
                        </a>
                    </li>
                
                    <li>
                        <a href="/tags">
                            
                                Tags
                            
                        </a>
                    </li>
                
                    <li>
                        <a href="https://github.com/WillWu12138/WillWu12138.github.io" target="_blank" rel="noopener">
                            
                                <i class="fa fa-github fa-stack-2x"></i>
                            
                        </a>
                    </li>
                
            </ul>
        </div>
        <!-- /.navbar-collapse -->
    </div>
    <!-- /.container -->
</nav>

    <!-- Main Content -->
    <!-- Page Header -->
<!-- Set your background image for this header in your post front-matter: cover -->

<header class="intro-header" style="background-image: url('http://www.codeblocq.com/assets/projects/hexo-theme-clean-blog/img/home-bg.jpg')">
    <div class="container">
        <div class="row">
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                <div class="post-heading">
                    <h1>PARTS</h1>
                    
                    <span class="meta">
                        <!-- Date and Author -->
                        
                        
                            2019-12-30
                        
                    </span>
                </div>
            </div>
        </div>
    </div>
</header>

<!-- Post Content -->
<article>
    <div class="container">
        <div class="row">

            <!-- Tags and categories -->
           
                <div class="col-lg-4 col-lg-offset-2 col-md-5 col-md-offset-1 post-tags">
                    
                        


<a href="/tags/Parts/">#Parts</a>


                    
                </div>
                <div class="col-lg-4 col-md-5 post-categories">
                    
                </div>
            

            <!-- Gallery -->
            

            <!-- Post Main Content -->
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                <h3 id="Characterization-of-Basic-Parts"><a href="#Characterization-of-Basic-Parts" class="headerlink" title="Characterization of Basic Parts"></a>Characterization of Basic Parts</h3><p>Several promoters with high expression level that are commonly used in Bacillus subtilis were screened to determine the most efficient ones for our experiments on the construction of biological parts. Three of such promoters, Groe (BBa_J100034), SecA (BBa_K1469002), P43 (BBa_K143013), exist in the iGEM registry already.</p>
<p>The primer sequences for individual promoters used for our synthetic biology project are as follows:</p>
<p>groe-up: CGCGGATCCCCAATACTGTTTTCTCAAATGGTATGTA</p>
<p>groe-down: CGGGGTACCTGAAATAACCTCCTCAATAGTATGA</p>
<p>YxiE-up: CGCGGATCCAATTGAAGCGCGCGAAGCCA</p>
<p>YxiE-down: CGGGGTACCGCTCTTCCCGCCTTTCGGACTG</p>
<p>SecA-up: CGCGGATCCGGACATCGTCCGTCAGAAACGCTTT</p>
<p>SecA-down: CGGGGTACCTCACACGCCTATTTTAGAGGCATGT</p>
<p>Ylbp-up: CGCGGATCCGTCACAACAGTCACGTCGTGATAAA</p>
<p>Ylbp-down: CGGGGTACCACATATATTGTAAACGCTTTATTTA</p>
<p>P43-up: CGCGGATCC TGATAGGTGGTATGTTTTCG</p>
<p>P43-down: CGGGGTACCTATAATGGTACCGCTATCACT</p>
<h3 id="Strains-and-Plasmids"><a href="#Strains-and-Plasmids" class="headerlink" title="Strains and Plasmids"></a>Strains and Plasmids</h3><p>The bacterium strain we used was B. subtilis WB600, and the plasmid was WB0911H-ASN (with the antibiotic ampicillin and kanamycin resistance cassette). Both the strain and plasmid were kindly donated to us by Professor Jian CHEN at Jiannan University, China.</p>
<p><img src="/2019/12/30/PARTS/1.png" alt="1"></p>
<p>Fig.1 pMAA0911H-ASN transformation vector</p>
<ol>
<li><p>PCR amplification of ASN gene product</p>
</li>
<li><p>Construction of transformation vectors (extraction of plasmids, digestion, conjugation and transformation, etc.)</p>
</li>
<li><p>Construction of Bacillus subtilis transformants</p>
</li>
<li><p>SDS-PAGE electrophoresis to confirm the ASN protein expression</p>
</li>
<li><p>The enzymatic activity of ASN was determined by colorimetric method. The detection process was divided into two steps: hydrolysis and coloration of ASN.</p>
<p> a.ASN hydrolysis: 100 ml diluted solution with ASN was added to the 1100 ml mixture of substrate and buffer. The reaction lasted for 10 min at 37°C. It was terminated by adding 100 ml of trichloroacetic acid (1.5 M). The mixture was then centrifuged at 12000 rpm for 2 minutes. The final system consisted of 900μL KH2PO4-K2HPO4 buffer (20 mM, pH 7.5) and 200 μl L-asparagine (189 mM).</p>
<p> b.Coloration: 100ml solution from ASN hydrolysis process was added to 3400μl deionized water, and 500μl Nessler’s reagent was added for coloration, the absorbance value was detected at 436 nm. (Here the definition of ASN Enzyme Activity Unit is: The amount of enzymes required to hydrolyze L-asparagine to release 1μM NH3 in 1 minute at 37°</p>
</li>
</ol>
<p>SDS-PAGE electrophoresis results: (band for the ASN protein at 43kDa is highlighted in red color)<br>Column M is the marker, column 1, 2, 3, 4, 5 and 6 are for the Bacillus subtilis constructs with the plasmids of WB0911H-ASN(original plasmid), WBGroE, WBYxiE, WBSecA, WBYlbP and WB43, respectively.</p>
<p><img src="/2019/12/30/PARTS/2.png" alt="2"></p>
<p>Fig.2 Gel Electrophoresis</p>
<p>ASN enzymatic activities with different transformants:</p>
<p><img src="/2019/12/30/PARTS/3.png" alt="3"></p>
<p>Fig.3 ANS enzymatic activity vs strain</p>
<h3 id="Conclusion"><a href="#Conclusion" class="headerlink" title="Conclusion"></a>Conclusion</h3><p>SDS-PAGE electrophoresis (Fig. 2) and the ASN enzymatic activity comparison (Fig. 3) show that the ASN production may be enhanced by the replacement of promoters. and among the promoters we have characterized, P43 has the highest start-up strength (the higher the enzyme activity, the thicker and brighter the corresponding band.</p>
<p>We have thus characterized the basic biobricks Groe (BBa_J100034), SecA (BBa_K1469002) and P43 (BBa_K208002).</p>
<h3 id="Construction-of-Composite-Parts"><a href="#Construction-of-Composite-Parts" class="headerlink" title="Construction of Composite Parts"></a>Construction of Composite Parts</h3><h3 id="Sequence-for-interval-peptide-and-proinsulin"><a href="#Sequence-for-interval-peptide-and-proinsulin" class="headerlink" title="Sequence for interval peptide and proinsulin:"></a>Sequence for interval peptide and proinsulin:</h3><p>In order to obtain higher proinsulin secretion, we have conducted the optimization of gene expression and proinsulin secretion using the yeast strain GS115 with pPICZαA plasmid as the expression vector. A interval peptide was introduced between the secretory signal peptides and proinsulin to improve gene expression for the proinsulin secretion.</p>
<p>The amino acid sequence of interval peptide is: EEAEAKR.<br>Nucleotide sequence is: GAAGAAGCTGAAGCTGAAGCTAAGAGA<br>The final sequence for interval peptide and proinsulin is:</p>
<p>GAAGAAGCTGAAGCTAAGAGAATGGCGCTTTGGATGAGACTGCTGCCGCTGCTGGCACTTCTTGCGCTTTGGGGACCGGACCCAGCAGCGGCATTTGTTAATCAACATCTTTGTGGCTCACATCTTGTGGAAGCACTTTATCTTGTTTGCGGAGAAAGAGGATTTTTCTATACACCGAAAACAAGAAGAGAAGCAGAAGATCTTCAGGTTGGACAGGTTGAACTGGGCGGCGGACCGGGCGCAGGATCACTTCAGCCGCTGGCACTGGAAGGCTCACTGCAAAAAAGAGGCATTGTGGAACAATGCTGTACAAGCATTTGCAGCCTTTATCAGCTGGAAAATTATTGCAATTAA</p>
<h3 id="Plasmids"><a href="#Plasmids" class="headerlink" title="Plasmids"></a>Plasmids</h3><p><img src="/2019/12/30/PARTS/4.png" alt="4"></p>
<p>Fig.4 Palsmid map</p>
<p>We have selected the restriction sites as follows:</p>
<p>At the 5’ end: EcoR I GAATTC</p>
<p>At the 3’ end: Kpn I GGTACC</p>
<p>Experimental Methods</p>
<ol>
<li><p>pFormula of growth media:</p>
<p> a.YPG medium: 2.0% tryptone, 1% yeast extract, 2% glycerol (solid medium with 1.5% agar).</p>
<p> b.Complete YPD medium: 2.0% tryptone, 1% yeast extract, 2% glucose.</p>
<p> c.Induction BMGY medium: 2.0% tryptone, 1% yeast extract, 1.34% YNB, 2% glycerol, 10mmol/L potassium phosphate (pH 6.0), biotin 4×10-5% (by aseptic filtration).</p>
<p> d.Induction BMMY medium: 2.0% tryptone, 1% yeast extract, 1.34% YNB, 0.5% methanol, 10mmol/L potassium phosphate (pH 6.0), biotin 4×10-5% (by sterile filtration).</p>
</li>
<li><p>Preparation of competent Pichia pastoris cells:<br>Single colony in the plate bearing Pichia pastoris GS115 was picked up and inoculated into 50 mL shaker’s flask containing 5 mL YPD medium. It was incubated overnight at 30°C and 200 r/min. It was then used as a seed to inoculate 100 mL YPD liquid in 500 mL shaker’s flask to grow at 30°C and 200 r/min till OD600 = 1.2-2.0. The P. pastoris growth liquid was centrifuged at 4°C for 5 minutes at 1600×g, and the supernatant was completely removed. The pellet was re-suspended with 10 mL D-Sorbitol (conc. 1mol/L). The suspension was centrifuged at 4°C for 5 min at 1600×g. The supernatant was removed. It was repeated for 5 times. Use 1000μL 1mol/L D-Sorbitol to make final suspension, put it into ice bath.</p>
</li>
<li><p>Electroporation of Pichia pastoris:<br>Put 80 μL competent Pichia pastoris cell suspension into a pre-cooled 0.2cm electroporation cup, add 20 μL linearized plasmid and mix it. The mixture was placed on ice for 5 minutes (or - 20°C for 1 minute). Put the cup into the electroporator for electroporation (voltage 2.0 kV, electric shock time 5 ms). Then immediately add 1 mL pre-cooled sterile 1 mol/L D-Sorbitol and rest for 60 minutes. The MD plate was coated with 200 μL competent Pichia pastoris cell suspension and placed in an oven at 30°C for 20-60 mins. The plate is cultured for 3-5 days.</p>
</li>
<li><p>Screening of multi-copy Pichia pastoris recombinants:<br>P. pastoris recombinants were randomly selected from the YPG-Zeocin (Bleomycin, a kind of antibiotic) plate with aseptic toothpicks. The recombinants were grown on the YPG-Zeocin plates with Zeocin concentration gradient (300 ug/mL, 600 ug/mL). The incubator was set at 30°C for 3-5 days. Colonies in different plates were examined daily. According to the quantitative dependence between the copy number of heterologous genes and the resistance of zeocin, it can be inferred that the cells grown only on the plates with low zeocin concentration are with low copy numbers, and the cells grown on the plate with high zeocin concentration are with high copy numbers.</p>
</li>
<li><p>Induced expression of Pichia pastoris recombinants:<br>Recombinant colonies on YPD plates with different copy numbers were selected (100, 300, and 600 ug/mL Zeocin, respectively). They were used to inoculate 100 mL BMGY medium in 1000 mL flasks, and put in a shaker for 16-24 h at 30°C. When the OD600 was 2-6, the culture was replaced with same volume BMMY medium for induction under aseptic condition. The culture was continued at 28°C for 120 hours. Methanol was added every 24 hours to make total concentration of methanol in the liquid to be 0.5%. After fermentation, the supernatant was precipitated with trichloroacetic acid (TCA). 50% TCA was added with 0.2% sodium deoxycholate. 25% TCA was mixed with 75% medium centrifugal supernatant (volume ratio) evenly and placed on ice for 30 min. It was then centrifuged (4℃, 20,000 xg, for 20 mins), the supernatant was removed, pellets were resuspended with 10% TCA. It was centrifuged again (the same condition). Remove the supernatant, suspend it with cold acetone (- 20 C) and centrifuge it (the same condition). Precipitation was dried by a blower and electrophoretic detection was carried out by SDS buffer solution.</p>
</li>
</ol>
<h3 id="Experimental-Results"><a href="#Experimental-Results" class="headerlink" title="Experimental Results:"></a>Experimental Results:</h3><ol>
<li>Screening of Pichia pastoris transformants:</li>
</ol>
<p><img src="/2019/12/30/PARTS/5.jpg" alt="5"></p>
<p>Fig.5 Gradient screening of the transformants</p>
<p>A:300 μg/mL Zeocin Screening | B: 600 μg/mL Zeocin Screening</p>
<p>It can be seen from Fig.5 that some Pichia pastoris may have been integrated with multiple copies of gene expression nuclei, they may grow faster, have larger colony diameter, and may express proinsulin genes more efficiently.</p>
<ol start="2">
<li>8 high-copy transformants were selected and cultured in shaking flask. Their insulin expression levels were detected. The results of protein electrophoresis are as follows: </li>
</ol>
<p><img src="/2019/12/30/PARTS/6.png" alt="6"></p>
<p>Fig.6 Electrophoresis for proinsulin expression. Number1-8 refers to samples from recombinant Pichia pastoris, and the control is the electrophoresis result of proinsulin. Dark bars of number 1-8 existed almost at the same place of the positive control, indicating that the expression product is proinsulin.</p>
<h3 id="Conclusion-1"><a href="#Conclusion-1" class="headerlink" title="Conclusion"></a>Conclusion</h3><p>It is found that the highest expression level of recombinant protein can reach 150 mg/L, accounting for about 40% of total secreted protein for our project (with novel insertion of an interval peptide). In comparison, the highest secretory expression of proinsulin was only 32 mg/L in Pichia pastoris transformants reported in the literature. The expression of proinsulin was increased by about 5 times after our synthetic biology work to modify the N terminal of the proinsulin.</p>
<h3 id="Improve"><a href="#Improve" class="headerlink" title="Improve"></a>Improve</h3><p>Prevent the appearance of restriction site.</p>
<p>Experimental Methods</p>
<ol>
<li>Construction of bacteria. Original HPI gene and modified HPI gene are both transformed into bacteria B. subtilis</li>
<li>Fermentation</li>
<li>Protein extraction(through mechanical attrition)</li>
<li>Protein Gel Electrophoresis</li>
</ol>
<p>Experimental Results</p>
<p>Protein electrophoresis:</p>
<p><img src="/2019/12/30/PARTS/7.png" alt="7"></p>
<p>Fig.7 The SDS-PAGE result of bacteria with unmodified HPI gene(BBa_K1328003).Trial 1 is negative control–the supernatant remained after the collection of WB800N. Trial 2-3 are supernatant of sample 1 and 2. Trial 4-6 are 10-times-diluted supertanants of trial 1-3.</p>
<p>Through the Fig.7 it is known that bacteria with original HPI gene secreted a small amount of proinsulin that is not enough to be perceived by naked eyes.</p>
<p><img src="/2019/12/30/PARTS/8.png" alt="8"></p>
<p>Fig.8 SDS-PAGE result of bacteria with codon-optimized and N-terminal-modified HPI gene(BBa_K3141001).Trial 1 is negative control–the supernatant remained after the collection of WB800N. Trial 2-3 are supernatant of sample 13 and 16. Trial 4-6 are 5-times-diluted supertanants of trial 1-3.</p>
<h3 id="Conclusion-2"><a href="#Conclusion-2" class="headerlink" title="Conclusion"></a>Conclusion</h3><p>The target protein bars existed in expected size, where proinsulin is about to shown–at 10.5kD to 14kD. Bacteria with modified HPI gene produced large amount of target protein(as shown in Fig. 8). As a result,it is found that the modification to HPI sequence increased the production of protein.</p>


                
            </div>

            <!-- Comments -->
            
                <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                    


                </div>
            
        </div>
    </div>
</article>

    <!-- Footer -->
    <hr />

<!-- Footer -->
<footer>
    <div class="container">
        <div class="row">
            <div class="col-lg-8 col-lg-offset-2 col-md-10 col-md-offset-1">
                <ul class="list-inline text-center">
                    

                    

                    
                        <li>
                            <a href="https://github.com/WillWu12138/WillWu12138.github.io" target="_blank">
                                <span class="fa-stack fa-lg">
                                    <i class="fa fa-circle fa-stack-2x"></i>
                                    <i class="fa fa-github fa-stack-1x fa-inverse"></i>
                                </span>
                            </a>
                        </li>
                    

                    

                    

                    
                </ul>
                <p class="copyright text-muted">&copy; 2020 Will Wu<br></p>
                <p class="copyright text-muted">Original Theme <a target="_blank" href="http://startbootstrap.com/template-overviews/clean-blog/">Clean Blog</a> from <a href="http://startbootstrap.com/" target="_blank">Start Bootstrap</a></p>
                <p class="copyright text-muted">Adapted for <a target="_blank" href="https://hexo.io/">Hexo</a> by <a href="http://www.codeblocq.com/" target="_blank">Jonathan Klughertz</a></p>
            </div>
        </div>
    </div>
</footer>


    <!-- After footer scripts -->
    
<!-- jQuery -->
<script src="//code.jquery.com/jquery-2.1.4.min.js"></script>

<!-- Bootstrap -->
<script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>

<!-- Gallery -->
<script src="//cdnjs.cloudflare.com/ajax/libs/featherlight/1.3.5/featherlight.min.js" type="text/javascript" charset="utf-8"></script>

<!-- Disqus Comments -->



</body>

</html>